docs/user/orion.rst

```diff
@@ -0,0 +1,92 @@
+.. _orion:
```

Comment:
I suggest not writing "something will do something" but rather "something does something"

E.g. Not: The button will open a new window; But: The button opens a new window